src/nvim/eval.c

```diff
@@ -426,7 +426,8 @@ static struct vimvar {
   {VV_NAME("oldfiles",         VAR_LIST), 0},
   {VV_NAME("windowid",         VAR_NUMBER), VV_RO},
   {VV_NAME("progpath",         VAR_STRING), VV_RO},
-  {VV_NAME("command_output",   VAR_STRING), 0}
+  {VV_NAME("command_output",   VAR_STRING), 0},
+  {VV_NAME("completed_item",   VAR_DICT), VV_RO},
```

Comment:
Given the new issue (https://github.com/neovim/neovim.github.io/issues/98), I think it should be added back.

> I think the comma is useful...

Yes, you're right I think.